controllers/patientsController.js

```diff
@@ -5,24 +5,32 @@ module.exports.newPatient = async (req, res) => {
 }
 
 module.exports.createNewPatient = async (req, res) => {
+  console.debug("createNewPatient - req.body", req.body);
   // todo input validation, protection against injections
-  const firstNames = req.body['firstnames'].trim()
+  const firstNames = req.body['firstNames'].trim()
   if (!firstNames || firstNames.length === 0) {
+    console.error("Invalide firstNames", firstNames);
```

Comment:
ne pas logger le nom ici, c'est du personal data, on veut la contenir le plus possible dans la DB et ne pas l'écrire partout. (pareil pour lastname et INE plus bas)

Edit : et puis il doit etre vide de toutes facons :)

Reply:
oui, je me suis permis d'ajouter le log ici pour logguer seulement le cas d'erreurs "étrange", mais ça retournera certainement undefined : plutôt ceci

```suggestion
    console.error("Invalide firstNames");
```